codalab/worker/main.py

```diff
@@ -193,12 +194,23 @@ def connect_to_codalab_server(server, password_file):
 
 def main():
     args = parse_args()
-    # This quits if connection unsuccessful
-    bundle_service = connect_to_codalab_server(args.server, args.password_file)
+
     # Configure logging
     logging.basicConfig(
         format='%(asctime)s %(message)s', level=(logging.DEBUG if args.verbose else logging.INFO)
     )
+
+    # Initialize sentry logging (this is a null-op if
+    # the CODALAB_SENTRY_INGEST_URL env var is not set).
+    initialize_sentry()
```

Comment:
Seems like it would be more explicit if it were something like:
```if use_sentry(): initialize_sentry()```
where `use_sentry` is something from sentry_util.py` that just does the simple check.
(Same for `load_sentry_data` below.)
That way you don't need the detailed comment and to maintain it.

Reply:
good idea, i'll change it to use that.